superchain/extra/addresses/sepolia/op.json

```diff
@@ -1,5 +1,6 @@
 {
   "AddressManager": "0x9bFE9c5609311DF1c011c47642253B78a4f33F4B",
+  "DisputeGameFactoryProxy": "0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1",
```

Comment:
Is this the only new address for fault proofs? I thought the portal also changed. Any contracts that have new implementations should also be notes in https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/implementations/networks/sepolia.yaml

Reply:
Portal is same address but a new ABI

Reply:
Unresolved this comment since we still need to update that linked [file](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/implementations/networks/sepolia.yaml) to be like this (sanity check my semvers + addresses)

```yaml
optimism_portal:
  2.5.0: 0x2D778797049FE9259d947D1ED8e5442226dFB589
  3.3.0: 0x215b3A2d1766616d1C4701f3D117348711BaAe93
dispute_game_factory:
  0.2.0: 0x1B3cfC1F692076538c9B53750EBc412042D48605
```